buildenv

```diff
@@ -7,19 +7,40 @@ export ZOPEN_TARBALL_URL="https://ftp.gnu.org/gnu/gettext/gettext-0.21.tar.gz"
 export ZOPEN_TARBALL_DEPS="curl gzip tar make m4 perl"
 
 export ZOPEN_GIT_URL="https://github.com/autotools-mirror/gettext.git"
-export ZOPEN_GIT_DEPS="git make tar m4 perl autoconf automake help2man makeinfo xz"
+export ZOPEN_GIT_DEPS="git make tar m4 perl autoconf automake help2man makeinfo xz ncurses"
 
-export ZOPEN_EXTRA_CFLAGS="-qnocsect" # required because of multiple lock.c CUs
-export ZOPEN_EXTRA_LDFLAGS=""
-
-export ZOPEN_CHECK="skip"
+export ZOPEN_EXTRA_CONFIGURE_OPTS='--disable-acl --disable-ncurses'
```

Comment:
do we need to disable ncurses if we're adding it as a dependency?

Reply:
we do need to disable ncurses - can't remember what that busted - so let me pull it from the dependency list and make sure it's ok

Reply:
oh. that's for git. so it hasn't been exercised. tar doesn't pull in ncurses